packages/vm/src/runBlock.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Can you disable eslint per-line? If we do this for the whole file then it is likely that (at least for me) `console.log ` leftovers get pushed to a PR.